user-management/create_users.py

```diff
@@ -0,0 +1,53 @@
+import csv
+import fire
+import os
+import pprint
+import sys
+
+# This code is necessary for the script to access essential functions from the toolbox directory
+current_dir = os.path.dirname(os.path.realpath(__file__))
+parent_dir = os.path.dirname(current_dir)
+sys.path.append(parent_dir)
+
+# Importing required helper functions from the toolbox directory
+from toolbox.get_access_token import get_access_token
+from toolbox.post_request import post_request
+
+
+def create_users(base_url, client_id, client_secret, input_file_path="./sample-files/create_users_input.csv", output_file_path="./sample-files/create_users_output.csv", email_verified=0):
+    try:
+        api_url = base_url + "/api/v1/users"
+        access_token = get_access_token(base_url, client_id, client_secret)
+
+        with open(input_file_path, newline='') as input_file:
+            users = csv.reader(input_file, delimiter=',')
+
+            with open(output_file_path, 'w') as output_file:
+                writer = csv.writer(output_file)
```

Comment:
```suggestion
        with open(input_file_path, newline='') as input_file:
            users = csv.reader(input_file, delimiter=',')

        with open(output_file_path, 'w') as output_file:
            writer = csv.writer(output_file)
```


apakah bisa kalau begini?
kalau yang sekarang kan 

```
with readfile:
  data = readfile.read()
  # writefile masih didalam readfile 🤔 
  with writefile:
    for datum in data:
      pass
```

Reply:
ohh ga bisa ya
karena `users` itu masih belum data, tapi csv.reader 🤔 

harus ditampung dulu

Reply:
betul harus ditampung dulu

Reply:
gini aja?

```
        input_file = open(input_file_path, newline='')
        users = csv.reader(input_file, delimiter=',')

        output_file = open(output_file_path, 'w')
        writer = csv.writer(output_file)
```

Reply:
sbb, tadi debugging agak panjang 🙏🏼 

sepertinya bisa begini

```python
with open(file) as f:
  data = list(csv.reader(vile))

data # udah jadi list jadi aman disini
```

Reply:
https://github.com/datasaur-ai/api-client-example/pull/21/files#r1422213432